src/components/upgrader/boxfishUpgrader.ts

```diff
@@ -310,24 +314,25 @@ export default class BoxfishUpgrader extends EventEmitter implements IDeviceUpgr
     return this._cameraManager.api.setProductInfo({ flash_boot_state: state });
   }
 
-  async initFlash(fsblMvcmd: Buffer, boxfishPkg: BoxfishPackage): Promise<any> {
+  async initFlash(fsblMvcmd: Buffer, boxfishUpgraderFile: IBoxfishUpgraderFile): Promise<any> {
```

Comment:
Is initFlash actually used? Probably not relevant for this review, but you're changing it, so I guess you own it?